README.md

```diff
@@ -3,11 +3,35 @@
 A key-value store for Slack
 
 ### Setup
-To use, add an outgoing webhook in Slack integrations pointing to `slack-slam.herokuapp.com/message` and make the trigger word `slam`
+
+#### Install app
+Clone this repo to your computer and deploy it as you would a Rails app.
+
+If you're using Heroku, Scalingo or such, change `config/database.yml`
+to uncomment the lines with `DATABASE_URL` and comment the other `production:` vars.
+Set your own `SECRET_KEY_BASE`.
+
```

Comment:
I'm not sure I understand.
Sure, this section is for deploying elsewhere, but I don't want to mess with your app ;)
About the next sections (Authorising, seeding) I think they were required and undocumented.
Or did I do something wrong?
I'll be glad to separate and INSTALL.md from the README.md if you think it's best!